src/modules/developer_tools.json

```diff
@@ -0,0 +1,250 @@
+{
+    "openrpc": "1.2.4",
+    "info": {
+        "title": "DeveloperTools",
+        "description": "A module for supporting off-device developer tools.",
+        "version": "0.0.0"
+    },
+    "methods": [
+        {
+            "name": "find",
```

Comment:
Wifi module has a method called `scan` that is quite similar. I'm not familiar with DNS-SD, so if `find` is explicitly meaningful it might make sense to use. But if it's just a reasonable name you picked, then it might be better to stay aligned with the WiFi module.

Reply:
think I agree with scan

Reply:
The various implementations and docs I've seen use "browse", "query", or "find". I have never seen "scan", ever.

It's also misleading, at least to me, because when I hear "scan" I think of "port scan", which is not what's happening... it's just a(n ongoing) DNS resolution.

That said, I'll change it to scan (and cancelScan) since you both prefer it.